### PR TITLE
feat: adds sorting of traces by node ID, with content ID marker

### PR DESCRIPTION
**Pull request by mrferris**
Implements #301 

Toggle-able via selector, with permanent memory of preferred selection.

Screenshot 2024-09-19 at 9 43 27 AM

### CODE REVIEW

**Comment by KolbyML** [comment]
Does the "should store" outline not work on this graph? From the picture in the PR it doesn't show the "should store" outline anywhere which is why I am asking

File: glados-web/assets/js/trace/main.js [hunk 1]
```diff
@@ -146,8 +147,10 @@ function createGraphData(trace) {
         metadata: {
             nodesContacted: nodes.length - 1,
             nodesResponded: Object.keys(responses).length - 1
-        }
+        },
+        contentId: '0x' + trace.targetId.map(byte => byte.toString(16).padStart(2, '0')).join(''),
     }
+    console.log(graph);
```
Comment (KolbyML) [inline glados-web/assets/js/trace/main.js:153]: Do we still need this debug log?

**Review by KolbyML — COMMENTED** [review]
Most things look good, I am interested in a comment if "should store" is broken or not, but other then that the PR looks very good

File: glados-web/assets/js/trace/main.js (hunk 1 above)
Comment (mrferris) [reply glados-web/assets/js/trace/main.js:153]: I removed it previously and have found that it's actually nice to have so I've added it back.

**Comment by mrferris** [comment]
@KolbyML It's missing from the screenshot because this is from a local instance in which I hadn't run any censuses, which is where the "should store" data comes from. "should store" still works, and it happens on the backend which isn't touched by this PR.

**Review by KolbyML — APPROVED** [review]
:shipit:

**Pull request merged**